lms/djangoapps/support/urls.py

```diff
@@ -64,5 +65,10 @@
         ProgramEnrollmentsInspectorView.as_view(),
         name='program_enrollments_inspector'
     ),
+    re_path(
+        r'get_saml_providers_with_org_key/?$',
```

Comment:
get_sample_provider is good enough. with_org gives away some implementation/selection details.